inc/ScaffoldPackageCommand.php

```diff
@@ -273,7 +273,7 @@ public function package_readme( $args, $assoc_args ) {
 				$readme_args['commands'][] = array(
 					'name' => "wp {$command}",
 					'shortdesc' => $parent_command['description'],
-					'synopsis' => "wp {$command} {$parent_command['synopsis']}",
+					'synopsis' => "wp {$command}" . ( empty( $parent_command['subcommands'] ) ? " {$parent_command['synopsis']}" : "" ),
```

Comment:
> Actually, I'm not exactly sure how to test this, or if it should be tested at all.

Ok, that's reasonable. If it's a small change, I'm typically fine not including tests if it's too hard to test for the change.